raiden/utils/__init__.py

```diff
@@ -53,6 +53,15 @@
 GLOBAL_CTX = secp256k1.lib.secp256k1_context_create(secp256k1.ALL_FLAGS)
 
 
+def safe_address_decode(address):
+    try:
+        address = address.decode('hex')
+    except TypeError:
```

Comment:
I guess the `pass` here assumes it's already in hexadecimal. What if it is in some other form totally wrong? Say an `int` or a `list`. Do we want to check it here or just let it fail somewhere else and throw an exception?